Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2522,7 +2522,7 @@ HRESULT Stage(CGContextRef context, ID2D1DeviceContext* deviceContext) override
             deviceContext->Clear({ 0, 0, 0, 0 }); // Clear the original target to transparent black.
             RETURN_IF_FAILED(inputBitmap.As(&copiedImage));
         }
-        return __super::Stage(context, deviceContext);
+        return __super ::Stage(context, deviceContext);
```

Comment:
Please fix the sad formatting before merge.

---
In reply to: [107024911](https://github.com/Microsoft/WinObjC/pull/2274#discussion_r107024911) [](ancestors = 107024911,107005905)